draft-pauly-httpbis-geoip-hint.md

```diff
@@ -24,36 +24,63 @@ author:
     name: David Schinazi
     organization: Google LLC
     email: dschinazi.ietf@gmail.com
-
+ -
+    ins: C. McMullin
+    name: Ciara McMullin
+    organization: Google LLC
+    email: ciaramcmullin@google.com
+ -
+    ins: D. Mitchell
+    name: Dustin Mitchell
+    organization: Google LLC
+    email: djmitche@gmail.com
 --- abstract
 
-This documents defines an HTTP Client Hint that allows a client to share information
-about its IP Geolocation. This helps ensure that servers have information about location
-that is consistent with what a client expects and what other servers use.
+The rise of IP-hiding technologies such as VPNs and proxies helps improve user
+privacy, but it also brings some drawbacks. Maintaining a geographically
+relevant user experience requires large pools of IP addresses, which can
+be costly. Additionally, users often receive inaccurate geolocation
+results because servers rely on geo-IP feeds that can be outdated. To
+address these challenges, we can allow clients to actively send their
+network geolocation directly to the origin server via an HTTP Client
+Hint. This approach will not only enhance geolocation accuracy and reduce IP
+costs, but it also gives clients more transparency regarding their perceived
+geolocation.
```

Comment:
```suggestion
Techniques that improve user privacy by hiding original client IP addresses,
such as VPNs and proxies, have faced challenges with server that rely on
IP addresses to determine client location. Maintaining a geographically
relevant user experience requires large pools of IP addresses, which can
be costly. Additionally, users often receive inaccurate geolocation
results because servers rely on geo-IP feeds that can be outdated. To
address these challenges, we can allow clients to actively send their
network geolocation directly to the origin server via an HTTP Client
Hint. This approach will not only enhance geolocation accuracy and reduce IP
costs, but it also gives clients more transparency regarding their perceived
geolocation.
```